hardwarelibrary/communication/sutter.py

```diff
@@ -0,0 +1,149 @@
+from struct import *
+import sys
+import serial
+from serial.tools.list_ports import comports
+#sys.path.insert(0, "/Users/justinemajor/Documents/gph.doc/git/PyHardwareLibrary/hardwarelibrary/communication")
+import time
+import array
+import os
+import serial
+
+#/dev/cu.usbserial-SI8YCLBE
+class SutterDevice:
+    def __init__(self):
+        """
+        SutterDevice represents a XYZ stage.  
+        """
+        
+        self.port = serial.Serial("/dev/cu.usbserial-SI8YCLBE", timeout=5, baudrate=128000)
+
+        #ajouter les instances nécessaires
+
+        self.microstepsPerMicrons = 16
+
+    def initializeDevice(self):
+        """
+        We do a late initialization: if the device is not present at creation, it can still be
+        initialized later.
+        """
+        if self.port is not None:
+            return
+        
+        self.port = serial.Serial("/dev/cu.usbserial-SI8YCLBE", timeout=5, baudrate=128000) 
```

Comment:
Remove hardcoded port.

Reply:
delete or replace by SerialPort (once I find it)?